src/System.ComponentModel.Primitives/src/System/ComponentModel/DescriptionAttribute.cs

```diff
@@ -52,17 +45,11 @@ public override bool Equals(object obj)
             }
 
             DescriptionAttribute other = obj as DescriptionAttribute;
-            return (other != null) && other.Description == Description;
+            return other?.Description == Description;
```

Comment:
This looks like a buggy change, e.g. what if this.Description is null and obj is null... then this will return true when it should return false.